features/appmgt/org.wso2.carbon.appmgt.publisher.feature/src/main/resources/publisher/apis/v1/validations_api_router.jag

```diff
@@ -1,96 +1,143 @@
 <%
 
 /*
-Description: The class is used to perform validations on a field name value pair
-            GET /api/validations/assets/{type}/{fieldName}/{fieldValue}     Returns a message object indicating
-                                                                            whether the field is correct
-Filename: validations_api_router.jag
-Created Date: 8/10/2013
-*/
-include("/extensions/webapp/modules/jagg/jagg.jag");
-
-require('/modules/publisher.js').exec(function(ctx) {
-
-var config = require('/config/publisher.json');
-var caramel = require('caramel');
-var utility=require('/modules/utility.js').rxt_utility();
-var apiProvider = jagg.module('manager').getAPIProviderObj();
-var router = require('/modules/router-g.js').router();
-var routeManager = new router.Router();
-
-var log = new Log();
+ Description: The class is used to perform validations on a field name value pair
+    GET /api/validations/assets/{type}/{fieldName}/{fieldValue}     Returns a message object indicating whether the field is correct
+ Filename: validations_api_router.jag
+ Created Date: 8/10/2013
+ */
 
-routeManager.register('GET', 'publisher', '/publisher/api/validations/assets/{type}/{fieldName}/{fieldValue}', function (context) {
-
-    if(context.params){
-      var shortName = context.params.type;
-      var fieldName = context.params.fieldName;
-      var fieldValue=context.params.fieldValue;
-    }
-	
-    try{
-
-        var contextVal = '/'+fieldValue;
-        var oldContextVal = '';
+include("/extensions/webapp/modules/jagg/jagg.jag");
```

Comment:
Why we depend on extensions jagg modules in API level file? We can have /jagg/jagg.jag module in publisher jaggery app root.